src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -198,6 +210,15 @@ interface RunningScript {
   ramUsage: number;
   /** Hostname of the server on which this script runs */
   server: string;
+  /** Properties of the tail window, or null if it is not shown */
+  tailProperties: TailProperties | null;
+  /**
+   * The title, as shown in the script's log box. Defaults to the name + args,
+   * but can be changed by the user. If it is set to a React element (only by
+   * the user), that will not be persisted, and will be restored to default on
+   * load.
+   */
+  title: string | React.ReactElement;
```

Comment:
Can you try regenerating docs? I have a feeling these references to React will not do well, either erroring out or just making bad .md output. Will be able to test this in a little while.

Reply:
Whoops, forgot to do that.

Looks like it comes out fine. (I am also mildly surprised.) ReactElement isn't a link, but renders OK.